config/webpack/webpack.config.js

```diff
@@ -70,6 +71,13 @@ module.exports = function (dirname, config = {}) {
             new CircularDependencyPlugin({
                 exclude: /(node_modules|examples)\/./,
                 failOnError: false // https://github.com/nodejs/readable-stream/issues/280#issuecomment-297076462
+            }),
+            new GlobalizePlugin({
+                production: false,
+                developmentLocale: "en",
+                supportedLocales: ["en", "es", "fr"],
+                messages: "../../src/i18n/[locale].json",
```

Comment:
The easiest solution is to let each extension to load messages. In `*-module` file they can do something like:
```ts
Globalize.loadMessages(require('../../i18n/en'));
Globalize.loadMessages(require('../../i18n/de'));
```
assuming that each extension has `i18n` folder next to `src`

If it works we can generate such code later.